SinaiStCatherine/SinaiNewEt001.xml

```diff
@@ -49,8 +49,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                       </dimensions>
                       <measure unit="g" type="weight">248</measure>
                     </extent>
-               <!--     <foliation>Foliation in the upper outer corner.</foliation>-->
+               <foliation>Foliation by Beta masaheft in the upper outer corner.</foliation>
```

Comment:
I would either not give the responsible or give the correct project name with diacritics

Reply:
ok